### PR TITLE
Add proto files

### PR DESCRIPTION
**Pull request by celestian**
label: architecture

https://stackoverflow.com/questions/13939904/how-to-use-protocol-buffers-with-autoconf-automake

### CODE REVIEW

File: src/demofile.cc [hunk 1]
```diff
@@ -165,6 +165,12 @@ void Demofile::print_game_details(void)
...
+              << "Game lenght (min): " << header.playback_time / 60 << std::endl
+              << "------------------------" << std::endl
+              << "networkprotocol:   " << header.networkprotocol << std::endl
+              << "gamedirectory:     " << header.gamedirectory << std::endl
+              << "playback_ticks:    " << header.playback_ticks << std::endl
+              << "playback_frames:   " << header.playback_frames << std::endl
+              << "signonlength:      " << header.signonlength
               << std::endl;
-}
+}
```
Comment (stlaz) [inline src/demofile.cc]: PLease add an EOL.

File: .gitignore [hunk 2]
```diff
@@ -53,3 +53,7 @@ stamp-h1
 *.dem
 *.dem.info
 data/match.txt
+
+# proto files
+*.pb.cc
+*.pb.h
```
Comment (stlaz) [inline .gitignore]: Please add an EOL.

File: src/cstrike15_gcmessages.proto [hunk 3]
```diff
@@ -0,0 +1,945 @@
+import "src/steammessages.proto";
```
Comment (stlaz) [inline src/cstrike15_gcmessages.proto]: Let's have a subfolder for the .proto files, e.g. `src/protobuffs`

**Review by stlaz — CHANGES_REQUESTED** [review]

**Comment by celestian** [comment]
Addressed and it works now :-)

**Pull request merged**